config/server-vars.yml

```diff
@@ -23,16 +23,70 @@ EDXAPP_PREVIEW_LMS_BASE: "{{ EDXAPP_LMS_BASE }}"
 #EDXAPP_ENABLE_THIRD_PARY_AUTH: true
 #EDXAPP_ENABLE_OAUTH2_PROVIDER: true
 
+edxapp_theme_name: 'default'
+
+EDXAPP_COUNTRIES_OVERRIDE:
+  BO: "Bolivia"
+  BQ: "Bonaire"
+  VG: "British Virgin Islands"
+  BN: "Brunei"
+  CD: "Congo (DRC)"
+  IR: null
+  CB: null
+  FK: "Falkland Islands"
+  VA: null
+  HK: "Hong Kong SAR"
+  XJ: "Jan Mayen"
+  KP: null
+  KR: "Korea"
+  XK: "Kosova"
+  LA: "Laos"
+  MO: "Macao SAR"
+  MK: "Macedonia, FYRO"
+  FM: "Micronesia"
+  MD: "Moldova"
+  PS: "Palestinian Authority"
+  PN: "Pitcairn Island"
+  RU: "Russia"
+  SX: "Sint Maarten"
+  XS: "Saba"
+  MF: "Saint Martin"
+  SH: "St Helena, Ascension, Tristan da Cunha"
+  SJ: "Svalbard"
+  SD: null
+  SY: null
+  TW: "Taiwan"
+  TZ: "Tanzania"
+  UM: "U.S. Outlying Islands"
+  VI: "U.S. Virgin Islands"
+  US: "United States"
+  GB: "United Kingdom"
+  VE: "Venezuela"
+  VN: "Vietnam"
+  EH: null
+ 
+EDXAPP_REGISTRATION_EXTRA_FIELDS: 
+  city: "hidden"
+  country: "required"
+  gender: "hidden"
+  goals: "optional"
+  honor_code: "required"
+  level_of_education: "hidden"
+  mailing_address: "hidden"
+  year_of_birth: "hidden"
+
 # Configure Open edX features
 # see: http://edx.readthedocs.org/projects/edx-installing-configuring-and-running/en/latest/feature_flags/feature_flag_index.html
 EDXAPP_FEATURES:
   #ENABLE_THIRD_PARTY_AUTH: "{{ EDXAPP_ENABLE_THIRD_PARTY_AUTH }}"
   #ENABLE_OAUTH2_PROVIDER: "{{ EDXAPP_ENABLE_OAUTH2_PROVIDER }}"
-  #ENABLE_SYSADMIN_DASHBOARD: true
+  ENABLE_SYSADMIN_DASHBOARD: true
   #ENABLE_INSTRUCTOR_EMAIL: true
   #REQUIRE_COURSE_EMAIL_AUTH: true
   PREVIEW_LMS_BASE: "{{ EDXAPP_PREVIEW_LMS_BASE }}"
   DISABLE_COURSE_CREATION: true
+  USE_CUSTOM_THEME: true
+  ALLOW_ALL_ADVANCED_COMPONENTS: true
 

```

Comment:
Since you're enabling other features, go ahead and uncomment/change:
* ENABLE_INSTRUCTOR_EMAIL
* REQUIRE_COURSE_EMAIL_AUTH
* EDXAPP_TIME_ZONE: "America/Los_Angeles"